pkg/disk/rpcserver/nodeserver.go

```diff
@@ -91,47 +92,44 @@ func (ns *NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 	stagePath := req.GetStagingTargetPath()
 	volumeId := req.GetVolumeId()
 
+	// check if volume exists in the first place
```

Comment:
remove `in the first place` as it's a bit misleading,  this is not the first check in this function.